pino.js

```diff
@@ -152,10 +154,12 @@ function asJson (obj, msg, num) {
       if ((notHasOwnProperty || obj.hasOwnProperty(key)) && value !== undefined) {
         value = this.stringify(this.serializers[key] ? this.serializers[key](value) : value)
         if (value !== undefined) {
-          data += ',"' + key + '":' + value
+          objData += ',"' + key + '":' + value
         }
       }
     }
+    this.objectKeyString === ',"":{' || Object.keys(obj).length < 1
```

Comment:
can you avoid calling Object.keys() for every passed object? We are already iterating over all the properties anyway, maybe we can set a boolean somewhere.

Reply:
It's really a check for `if (objData.length > 0) ...`